### PR TITLE
Add Quickswap v2 to dex.trades

### PR DESCRIPTION
**Pull request by darvinrio**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/quickswap/polygon/quickswap_polygon_schema.yml [hunk 1]
```diff
@@ -96,12 +96,61 @@ models:
...
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: polygon
+          project: quickswap
+          version: 2
```
Comment (Hosuke) [inline models/quickswap/polygon/quickswap_polygon_schema.yml:122]: Hi @darvinrio,
in order for `check_dex_seed` to work, you may need to add some seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/quickswap/polygon/quickswap_polygon_schema.yml (hunk 1 above)
Comment (darvinrio) [reply models/quickswap/polygon/quickswap_polygon_schema.yml:122]: @Hosuke added the request seed records

File: models/prices/polygon/prices_polygon_tokens.sql [hunk 2]
```diff
@@ -60,6 +60,9 @@ FROM
     ("revv-revv", "polygon", "REVV", "0x70c006878a5a50ed185ac4c87d837633923de296", 18),
     ("grain-granary","polygon","GRAIN","0x8429d0afade80498eadb9919e41437a14d45a00b",18),
     ("oath-oath","polygon","OATH","0xc2c52ff5134596f5ff1b1204d3304228f2432836",18),
-    ("tetu-tetu-reward-token","polygon","TETU","0x255707B70BF90aa112006E1b07B9AeA6De021424",18)  
-
+    ("tetu-tetu-reward-token","polygon","TETU","0x255707B70BF90aa112006E1b07B9AeA6De021424",18),
+    ("quick-quickswap","polygon","QUICK","0x831753dd7087cac61ab5644b308642cc1c33dc13",18),
```
Comment (Hosuke) [inline models/prices/polygon/prices_polygon_tokens.sql:64]: API: https://api.coinpaprika.com/v1/coins/quick-quickswap

File: models/prices/polygon/prices_polygon_tokens.sql [hunk 3]
```diff
@@ -60,6 +60,9 @@ FROM
...
+    ("dimo-dimo","polygon","DIMO","0xe261d618a959afffd53168cd07d12e37b26761db",18),
```
Comment (Hosuke) [inline models/prices/polygon/prices_polygon_tokens.sql:65]: API: https://api.coinpaprika.com/v1/coins/dimo-dimo

File: models/prices/polygon/prices_polygon_tokens.sql [hunk 4]
```diff
@@ -60,6 +60,9 @@ FROM
...
+    ("gddy-giddy","polygon","GDDY","0x67eb41a14c0fe5cd701fc9d5a3d6597a72f641a6",18),
```
Comment (Hosuke) [inline models/prices/polygon/prices_polygon_tokens.sql:66]: API: https://api.coinpaprika.com/v1/coins/gddy-giddy

File: models/prices/polygon/prices_polygon_tokens.sql [hunk 5]
```diff
@@ -60,6 +60,9 @@ FROM
...
+    ("amaticc-ankr-reward-earning-matic","polygon","aMATICc","0x0e9b89007eee9c958c0eda24ef70723c2c93dd58",18)
```
Comment (Hosuke) [inline models/prices/polygon/prices_polygon_tokens.sql:67]: API: https://api.coinpaprika.com/v1/coins/amaticc-ankr-reward-earning-matic

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @darvinrio 

- Metadata & naming conventions ✅
- Config blocks match uniqueness tests in schema files ✅
- Tests and seeds added for the new models ✅
- Incremental filters added when joining large tables ✅
- Contract addresses are lowercase ✅
- Inner join to large tables(*.transactions) as possible ✅
- `CAST()` used in the base models ✅

[Sample quickswap_v2_polygon_trades](https://dune.com/queries/2459572)

File: models/prices/polygon/prices_polygon_tokens.sql (hunk 5 above)
Comment (Hosuke) [reply models/prices/polygon/prices_polygon_tokens.sql:67]: Currently this token is not active.

Comment (darvinrio) [reply models/prices/polygon/prices_polygon_tokens.sql:67]: i ll remove it then

**Comment by Hosuke** [comment]
GM @augustog, this PR is ready for final review.

**Comment by darvinrio** [comment]
@Hosuke @augustog any updates on this ?

**Comment by Hosuke** [comment]
> @Hosuke @augustog any updates on this ?

This PR is in the merging queue, please be patient. 
It will be merged soon, thanks.

**Comment by darvinrio** [comment]
> 

thanks for the update. 💛

**Review by jeff-dude — APPROVED** [review]
thanks all

**Pull request merged**